### PR TITLE
Update of Material UI

### PR DESCRIPTION
**Pull request by rtaraszka**
Updated components, configuration and Circle CI config.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
   "name": "material-ui",
   "version": "1.0.0",
-  "description": "",
+  "description": "Material UI integrated with UXPin Merge",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "uxpin-merge --webpack-config ./webpack.config.js --wrapper ./src/Wrapper/UXPinWrapper.js"
+    "push": "uxpin-merge push --webpack-config ./webpack.config.js --wrapper ./src/Wrapper/UXPinWrapper.js",
```
Comment (michalstocki) [inline package.json]: You can move `webpackConfig` and `wrapper` to `uxpin.config.js`

File: src/Card/Card.js [hunk 2]
```diff
@@ -11,12 +11,17 @@ function Card(props) {
 }
 
 Card.propTypes = {
+
+  /** @uxpinignoreprop */
```
Comment (michalstocki) [inline src/Card/Card.js]: What's the point to ignore the `children` property? Doesn't it block dropping other components into Card?

File: src/FormLabel/FormLabel.js [hunk 3]
```diff
@@ -13,18 +13,21 @@ FormLabel.propTypes = {
...
   /**
+   * @uxpinignoreprop 
    * @ignore
    */
   className: PropTypes.string,
 
   /**
-   * The component used for the root node.
+   * 
+   * @uxpinignoreprop The component used for the root node.
```
Comment (michalstocki) [inline src/FormLabel/FormLabel.js]:
```suggestion
   * @uxpinignoreprop
   * The component used for the root node.
```

File: src/Card/Card.js (hunk 2 above)
Comment (rtaraszka) [reply src/Card/Card.js]: http://recordit.co/HnXn6aJc5N

You can still nest CC, but there is no `children` property on right panel (that way we block the possibility to break the CC)

Comment (michalstocki) [reply src/Card/Card.js]: I didn't knew the trick

File: src/FormLabel/FormLabel.js (hunk 3 above)
Comment (rtaraszka) [reply src/FormLabel/FormLabel.js]: Updated

File: package.json (hunk 1 above)
Comment (rtaraszka) [reply package.json]: Updated

Comment (michalstocki) [reply package.json]: You can move remove now those flags also from the `push` command

Comment (rtaraszka) [reply package.json]: Updated

**Pull request merged**